tests/integration/linux_bridge_test.py

```diff
@@ -537,6 +539,37 @@ def test_disable_vlan_filtering_on_bridge(
             assertlib.assert_state_match(desired_state)
 
 
+def test_bridge_with_unmanaged_ports():
+    bridge_config = _create_bridge_subtree_config([])
+    with linux_bridge(TEST_BRIDGE0, bridge_config):
+        with dummy0_as_slave(TEST_BRIDGE0):
+            current_state = show_only((TEST_BRIDGE0,))
+            bridge_state = current_state[Interface.KEY][0]
+            port_subtree = bridge_state[LinuxBridge.CONFIG_SUBTREE].get(
+                LinuxBridge.PORT_SUBTREE, []
+            )
+
+            assert len(port_subtree) == 1
```

Comment:
Please assert the content of `port_subtree` instead of length.

Reply:
Done, thanks!